translator/src/main/java/org/finos/calmtranslator/translators/K8sModelTranslator.java

```diff
@@ -1,21 +1,89 @@
 package org.finos.calmtranslator.translators;
 
-import io.fabric8.kubernetes.api.model.Pod;
-import io.fabric8.kubernetes.api.model.PodBuilder;
+import io.fabric8.kubernetes.api.model.Namespace;
+import io.fabric8.kubernetes.api.model.NamespaceBuilder;
+import io.fabric8.kubernetes.api.model.Service;
+import io.fabric8.kubernetes.api.model.ServiceBuilder;
+import io.fabric8.kubernetes.api.model.ServicePort;
 import io.fabric8.kubernetes.client.utils.Serialization;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
 import org.finos.calmtranslator.calm.Core;
+import org.finos.calmtranslator.calm.Node;
+import org.finos.calmtranslator.calm.RelationshipType;
+
 public class K8sModelTranslator implements ModelTranslator<String> {
-	@Override
-	public String translate(final Core calmModel) {
-
-		Pod p = new PodBuilder().withNewMetadata()
-				.withName("excj496")
-				.endMetadata()
-				.build();
-		// Get YAML string
-		System.out.println(Serialization.asYaml(p));
-		// Get JSON string
-		System.out.println(Serialization.asJson(p));
-		return null;
-	}
+    @Override
+    public String translate(final Core calmModel) {
```

Comment:
I would return a list of K8s Components over a string. 
Then let the Spring serialiser in the controller deal with converting it to Yaml or JSON to return to the user.

Additionally this will make your tests significantly easier

Reply:
We can defo start with a list of strings - this also ensure order. I'll work on those changes.

Future state we might need another plan as i can see use providing two options

- flat output as yaml/json
- zip/tar containing a structured folder with files for each manifest

 But that can be a problem for future Matt ;)

Reply:
actually returning  `List<KubernetesResource>` works well